docs/10-WebServices.md

```diff
@@ -19,21 +19,43 @@ Configure modules in `api.suite.yml`:
 ``` yaml
 class_name: ApiTester
 modules:
-    config:
-		- REST:
-		    url: http://serviceapp/api/v1/
-		    depends: PhpBrowser
+    enabled:
+				- REST:
+				    url: http://serviceapp/api/v1/
+				    depends: PhpBrowser
 ```
 
-The REST module will automatically connect to `PhpBrowser`. In case you provide it with Symfony2, Laravel4, Zend, or other framework module, it will connect to them as well. Don't forget to run the `build` command once you finished editing configuration.
+The REST module will connect to `PhpBrowser` according to this configuration. Depending on web service we may deal with XML or JSON responses. Codeception handles well both data formats, however If you don't need one of them, you can explicitly specify JSON or XML parts of module will be used:
```

Comment:
Codeception handles **~~well~~** both data formats **well**

you can explicitly specify **that the** JSON or XML parts of **the** module will be used: